projects/ion/src/lib/button/button.component.ts

```diff
@@ -12,6 +12,7 @@ type Size = 'sm' | 'md' | 'lg' | 'xl';
 
 export interface IonButtonProps {
   label?: string;
+  title?: string;
```

Comment:
change to tooltip, because when tooltip is done we change only HTML. What do you think?

Reply:
good idea